src/test/java/org/fcrepo/auth/webac/WebACRolesProviderTest.java

```diff
@@ -126,13 +126,13 @@ public void noAclTest() throws RepositoryException {
 
         when(mockResource.getPath()).thenReturn(accessTo);
         when(mockResource.getContainer()).thenReturn(mockParentResource);
-        when(mockResource.getTriples(anyObject(),
-                    eq(PropertiesRdfContext.class))).thenReturn(new RdfStream());
+        when(mockResource.getTriples(anyObject(), eq(PropertiesRdfContext.class)))
```

Comment:
nope, just reformatted to make the line consistent with everything else in the file. shall I revert that?